int-tests/src/test-integration/java/io/neow3j/protocol/TransactionBuilderIntegrationTest.java

```diff
@@ -38,10 +39,11 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertThrows;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
+@Testcontainers
```

Comment:
I think this is the correct case to use this annotation since you have a test container as a member of the class.